api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -979,12 +979,13 @@ public CohortDefinition getNumberOfPatientsWhoAbandonedArtDuringPreviousMonthFor
     SqlCohortDefinition cd = new SqlCohortDefinition();
     cd.setName("Number of patients who Abandoned the ART during the current month");
     cd.addParameter(new Parameter("location", "Location", Location.class));
-    cd.addParameter(new Parameter("onDate", "On Date", Date.class));
+    cd.addParameter(new Parameter("onODate", "End Date", Date.class));
```

Comment:
Hi, there could be a typo on the parameter onDate.

Reply:
There is a typo. I suggest using `onOrBefore` as the name (also update query).